docs/kb/semgrep-secrets/per-product-ignore-not-working.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Product specific path ignores requires a supported CLI version
+tags: 
+    - Semgrep Secrets
+---
+
+# Why didn't Semgrep Secrets ignore the files/folders I specified?
+
+The Semgrep AppSec platform allows you to [define ignore patterns](https://semgrep.dev/docs/ignoring-files-folders-code#define-ignored-files-and-folders-in-semgrep-appsec-platform) for specific products within a project. However, these product-specific ignores for Semgrep Secrets require a CLI version `1.71.0` or later.
+
+If you are using an older version of the CLI, the path ignores for Semgrep Secrets will not be applied. Instead, the system will fallback to the SAST + SCA path ignores.
```

Comment:
```suggestion
If you use an older version of Semgrep, the path ignores for Semgrep Secrets that are set in Semgrep AppSec Platform are not applied. Instead, the system applies the path ignores from SAST and SCA to your Secrets scan as well.
```

@salolivares 
1. Regarding the last bit, I am not sure at all if I wrote it correctly. There are 2 ways to interpret the current wording: Secrets scans don't ignore any path at all, or Secrets scans fallback to using the ignores set by SAST & SCA. 
2. I changed the tense to simple present. This is purely a [technical writing convention](https://developers.google.com/style/tense)

Reply:
1. It's this one: Secrets scans fallback to using the ignores set by SAST & SCA
2. Awesome!

@s-santillan